gnc/sub8_controller/include/sub8_controller/pd_controller.h

```diff
@@ -47,8 +48,8 @@ class PDController {
   // Control Parameters
   unsigned int trans_history_length = 75;
   unsigned int angle_history_length = 75;
-  controller_deque translation_error_history;
-  controller_deque orientation_error_history;
+  ControllerDeque translation_error_history;
+  ControllerDeque orientation_error_history;
 
   // TODO: Make these ROSParams
   double kp_trans = 30;
```

Comment:
In the future, there should be a tunable gain for each degree of freedom, like [kp_x, kp_y, kp_z], [ki_x, ki_y ...
These gains will be in body frame so that we can tune for performance differences like the difficulty in strafing compared to moving forward.

Something along the lines of
```
k_world = R * diag(k) * R.T
```
will be needed to put them in world frame so they can be used like
```
controller_output = k_world * error
```